src/api/remote/git/repository.rs

```diff
@@ -342,7 +342,6 @@ fn parse_ls_remote(output: String) -> Vec<Reference> {
         .collect()
 }
 
-#[tracing::instrument]
 fn line_to_git_ref(line: &str) -> Option<Reference> {
```

Comment:
This was causing A LOT of traces. The function above (`parse_ls_remote`) calls this function for every single reference in the repo, and that can be a lot (tensorflow has 44,567 for example). I don't think we need a span created for each of these.